extensions/binding/org.eclipse.smarthome.binding.onewire/ESH-INF/thing/common.xml

```diff
@@ -17,12 +17,13 @@
 			<parameter name="resolution" type="text">
 				<label>Sensor resolution</label>
 				<options>
+					<option value="">default</option>
```

Comment:
This looks like a workaround to support sensors other than DS18B20. Would it make sense to have this parameter on the thing-type itself and provide a separate thing-type for the DS18B20? Or provide a separate channel-type w/o this option for _other_ temperature things?

Reply:
This is the reason why I implemented it slightly different in second attempt https://github.com/eclipse/smarthome/pull/6413
However I still think that configuration should be simpler, where every (or most of) device is `sensor`, something like that
```
sensors.add(DS1420, channels(ID));
sensors.add(DS1923, channels(ID, TEMPERATURE, HUMIDITY));
sensors.add(DS18B20, channels(ID, TEMPERATURE_HIRES));
sensors.add(DS18S20, channels(ID, TEMPERATURE));
```